docs/development-workflows/netlify-cms.md

```diff
@@ -0,0 +1,10 @@
+# Netlify CMS
```

Comment:
Netlify CMS is now Decap CMS. I still call it Netlify personally, but it's probably good to start switching over: https://decapcms.org/

Reply:
Yeah, that idea had crossed my mind, but I think it might make sense to do that as part of the "upgrade" / migration to Decap, since the version of the tool we use is called Netlify still, which could cause some confusion for what people see as branding in the admin.

Maybe this is just a good motivator to hurry up with that upgrade 😄 I'll create a ticket!